src/printer.js

```diff
@@ -4173,7 +4173,9 @@ function printAssignment(
 
   const canBreak =
     (isBinaryish(rightNode) && !shouldInlineLogicalExpression(rightNode)) ||
-    ((leftNode.type === "Identifier" || leftNode.type === "MemberExpression") &&
+    ((leftNode.type === "Identifier" ||
+      leftNode.type === "Literal" ||
```

Comment:
Literal only works for flow. You need to use isLiteral if you want it to work on babylon